js/registrationForm.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
+    <script src="registrationForm.js"></script>
+    <title>Registration form</title>
+</head>
+<body>
+    <form action="registrationForm.php" onsubmit="return validate()" method="POST">
+        <label for="username">Username:</label>
```

Comment:
You are missing validations on the html inputs. You could check here for a reference: https://github.com/martinely95/WEB2020-php/blob/master/ex5/index.html

Reply:
Giving feedback to the user as early as possible is always better.